proto/src/determined/rbac/v1/rbac.proto

```diff
@@ -85,6 +85,9 @@ enum PermissionType {
   // If assigned at a workspace scope, can only assign roles to that workspace
   // scope.
   PERMISSION_TYPE_ASSIGN_ROLES = 6002;
+
+  // Permission used on client to expect open source product behavior.
+  PERMISSION_TYPE_OSS_USER = -1;
```

Comment:
I'm somewhat concerned about adding this to the API definition.

From the API perspective we are never going to return or want to accept this ```OSS_USER``` permission. In the future when we allow customizable roles someone using our API will see this as an option.

Reply:
Hmm. I want to have this enum be the source of permissions for client-side permissions-check in `usePermissions` and avoid declaring them twice. It's not possible to extend a typescript enum. I might be able to replace it in types.ts with a type which is either `Api.PermissionType | ClientPermissionType`.

Reply:
Let me know if this alternative makes sense as committed ( https://github.com/determined-ai/determined/pull/5121/commits/da7238c50a272a228aede388da9851ae3111dd5f ) - could also go with PermissionType / MockPermissionType or other names for these types.